NEWS

```diff
@@ -1,5 +1,15 @@
 systemd System and Service Manager
 
+CHANGES WITH 248:
+
+        * The MountAPIVFS= service file setting now additionally mounts a tmpfs
+          on /run if it is not already a mount point. A writable /run has always
```

Comment:
doesn't matter, but we try to suffix dirs with / in documentation, as a hint they are dirs. we aren't particularly good at following through with this, but i'd still prefer if this would say `/usr/` rather than just `/usr`.

(It's like the thing where we suffix settings with `=` in docs, to indicate they are settings)